tools/IceRpc.Protobuf.Tools/IceRpc.Protobuf.Tools.props

```diff
@@ -16,7 +16,7 @@
                  the source build. -->
             <PropertyGroup>
                 <IceRpcProtocPath>$(MSBuildThisFileDirectory)obj/tools/</IceRpcProtocPath>
-                <IceRpcProtocGenPath>$(MSBuildThisFileDirectory)../IceRpc.ProtocGen/bin/$(Configuration)/net8.0/</IceRpcProtocGenPath>
+                <IceRpcProtocGenPath>$(MSBuildThisFileDirectory)../IceRpc.ProtocGen/bin/$(Configuration)/$(IceRpcTargetFramework)/</IceRpcProtocGenPath>
```

Comment:
It's not clear to me where IceRpcTargetFramework comes from. I don't see any import or Directory.Build.props with imports.

We should not count on the user of this props file to somehow import IceRpc.Version.props first.

Reply:
We don't, there are two branches here, and we count on `IceRpcTargetFramework` to be defined with the source builds.

For example, with a source build this condition is true:
```
<When Condition="Exists('$(MSBuildThisFileDirectory)../IceRpc.ProtocGen')">
````

For builds that use the NuGet package, everything is in the package's tools directory and there are no framework directories.

Reply:
Added comments to clarify this section of the props file